examples/bri-3/src/bri/identity/bpiSubjects/subjects.profile.ts

```diff
@@ -0,0 +1,54 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { createMap, forMember, mapFrom, Mapper, namingConventions, PascalCaseNamingConvention, SnakeCaseNamingConvention } from '@automapper/core';
+import { Injectable } from '@nestjs/common';
+import { BpiSubject } from './models/bpiSubject';
+import { BpiSubjectDto } from './api/dtos/response/bpiSubject.dto';
+
+@Injectable()
+export class SubjectsProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper) => {
+
+      createMap(
+        mapper,
+        BpiSubject,
+        BpiSubjectDto,
+        forMember(
+          (d) => d.publicKey,
+          mapFrom((s) => s.publicKey),
+        ),
+      );
+
+      createMap(
+        mapper,
+        BpiSubject,
+        BpiSubject,
+        forMember(
```

Comment:
just one question, i think in .net version of automapper 1:1 mapping are not required in profile, are they required here? nothing against it, just to be sure if we can reduce bunch of boilerplate code

Reply:
that was some PoC test code. I will remove this snippet.

Reply:
Yes, we only add forMember for mappings that are not 1:1